doc/modules/pipeline.rst

```diff
@@ -93,6 +92,14 @@ This is particularly important for doing grid searches::
     ...               svm__C=[0.1, 10, 100])
     >>> grid_search = GridSearchCV(clf, param_grid=params)
 
+Individual steps may also be replaced as parameters, and non-final steps may be
+ignored by setting them to ``None``::
+
+    >>> from sklearn.linear_model import LogisticRegression
+    >>> params = dict(reduce_dim=[None, PCA(5), PCA(10)],
+    ...               clf=[SVC(), LogisticRegression()],
```

Comment:
Did you mean `svm=[SVC(), LogisticRegression()]`? We should rename `svm` to `classifier` or maybe more explicitly `classifier_with_C`